js/controller.js

```diff
@@ -823,6 +844,17 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.DISCOVERY_API.SEND_DISPLAY_EVENT, eventData);
     },
 
+    sendVideoQualityChangeEvent: function(selectedContentData) {
+      this.state.videoQualityOptions.selectedBitrate = {
+        "id": selectedContentData.id
+      };
+      this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+      this.renderSkin();
+      this.mb.publish(OO.EVENTS.PAUSE);
+      this.mb.publish(OO.EVENTS.SET_EMBED_CODE, this.state.assetId);
+      this.mb.publish(OO.EVENTS.BITRATE_CHANGED, selectedContentData.selectedBitrate);
```

Comment:
out of the events we have in constants, SET_TARGET_BITRATE seems to be more appropriate